### PR TITLE
Allow custom icons

### PR DESCRIPTION
**Pull request by mmcc**
~~The new, custom icon is busted, so don't pull this in until we fix that.~~ The other piece of this PR is to allow a custom path to be set per icon, so if most of your icons come from one place you can specify a global `root-dir`, but set a `root-dir` for any icons in another set or location. 

**Edit:** Ok, added an audio-description.svg file that looks passable.

Also, we should think about ignoring the built files.

### CODE REVIEW

File: lib/grunt.js [hunk 1]
```diff
@@ -40,10 +46,13 @@ module.exports = function(grunt) {
       htmlTemplate: './templates/html.hbs',
       html: true,
       rename: function(path) {
-        path = path.replace(svgRootDir, '');
+        // We just want the filename, so split on backslashes and pop the last one.
+        let fileName = path.split('/').pop();
```
Comment (dmlap) [inline lib/grunt.js]: [basename](https://nodejs.org/api/path.html#path_path_basename_p_ext) is a bit more robust for this sort of thing.

File: lib/grunt.js [hunk 2]
```diff
@@ -40,10 +46,13 @@ module.exports = function(grunt) {
...
 
         let iconName = _.result(_.find(icons, function(icon) {
-          return icon.svg === path;
+          // We need to do the same as above so we can compare apples to apples
+          let svgName = icon.svg.split('/').pop();
```
Comment (dmlap) [inline lib/grunt.js]: basename again.

File: scss/_icons.scss [hunk 3]
```diff
@@ -7,8 +7,8 @@ $icon-font-path: '../fonts';
...
...h0hAWgCNwIPCBErGSQ0EgYEEjAcITYVAg8IESsZJDQSBgQSMBwhNhUCDwgRKxkkNBIGBBIwHCE2FxwBHd9ORwAAAAAQAMYAAQAAAAAAAQAHAAAAAQAAAAAAAgAHAAcAAQAAAAAAAwAHAA4AAQAAAAAABAAHABUAAQAAAAAABQALABwAAQAAAAAABgAHACcAAQAAAAAACgArAC4AAQAAAAAACwATAFkAAwABBAkAAQAOAGwAAwABBAkAAgAOAHoAAwABBAkAAwAOAIgAAwABBAkABAAOAJYAAwABBAkABQAWAKQAAwABBAkABgAOALoAAwABBAkACgBWAMgAAwABBAkACwAmAR5WaWRlb0pTUmVndWxhclZpZGVvSlNWaWRlb0pTVmVyc2lvbiAxLjBWaWRlb0pTR2VuZXJhdGVkIGJ5IHN2ZzJ0dGYgZnJvbSBGb250ZWxsbyBwcm9qZWN0Lmh0dHA6Ly9mb250ZWxsby5jb20AVgBpAGQAZQBvAEoAUwBSAGUAZwB1AGwAYQByAFYAaQBkAGUAbwBKAFMAVgBpAGQAZQBvAEoAUwBWAGUAcgBzAGkAbwBuACAAMQAuADAAVgBpAGQAZQBvAEoAUwBHAGUAbgBlAHIAYQB0AGUAZAAgAGIAeQAgAHMAdgBnADIAdAB0AGYAIABmAHIAbwBtACAARgBvAG4AdABlAGwAbABvACAAcAByAG8AagBlAGMAdAAuAGgAdAB0AHAAOgAvAC8AZgBvAG4AdABlAGwAbABvAC4AYwBvAG0AAAACAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABUAAAECAQMBBAEFAQYBBwEIAQkBCgELAQwBDQEOAQ8BEAERARIBEwEUARUEcGxheQtwbGF5LWNpcmNsZQVwYXVzZQt2b2x1bWUtbXV0ZQp2b2x1bWUtbG93CnZvbHVtZS1taWQLdm9sdW1lLWhpZ2gQZnVsbHNjcmVlbi1lbnRlcg9mdWxsc2NyZWVuLWV4aXQGc3F1YXJlB3NwaW5uZXIJc3VidGl0bGVzCGNhcHRpb25zCGNoYXB0ZXJzBXNoYXJlA2NvZwZjaXJjbGUOY2lyY2xlLW91dGxpbmUTY2lyY2xlLWlubmVyLWNpcmNsZRFhdWRpby1kZXNjcmlwdGlvbgAAAAAA) format('truetype');
```
Comment (dmlap) [inline scss/_icons.scss]: Why do we have the ttf here but not for videojs-icons.css above?

Comment (mmcc) [inline scss/_icons.scss]: Hrm there's ttf above too, it's just post-sass so there's no line break between them.

File: lib/grunt.js [hunk 4]
```diff
@@ -39,11 +46,15 @@ module.exports = function(grunt) {
       htmlDest: 'preview.html',
       htmlTemplate: './templates/html.hbs',
       html: true,
-      rename: function(path) {
-        path = path.replace(svgRootDir, '');
+      rename: function(iconPath) {
+        // We just want the filename, so split on backslashes and pop the last one.
```
Comment (dmlap) [inline lib/grunt.js]: This in in the category of pedantic nitpicks but now this comment is inaccurate.

**Comment by dmlap** [comment]
LGTM

**Pull request merged**